Sources/main.swift

```diff
@@ -102,6 +102,11 @@ routes.add(method: .post, uri: "/webhook", handler: { request, response in
       if let titleLabel = PRLabelAnalysis.getTitleLabel(title: issueData.title) {
         labelsToAdd.append(titleLabel)
       }
+      let actionabilityLabel = "Needs actionability review"
```

Comment:
Should I move this to `ProcessInfo.processInfo.environment`?

Reply:
If so, where do I define it?

Reply:
yeah, we should be able to define it but currently can't. I have opened an issue for it here: #18